ocean_lib/http_requests/requests_session.py

```diff
@@ -18,13 +18,13 @@ def get_requests_session() -> Session:
     session.mount(
         "http://",
         HTTPAdapter(
-            pool_connections=25, pool_maxsize=25, pool_block=True, max_retries=1
+            pool_connections=25, pool_maxsize=25, pool_block=True, max_retries=3
```

Comment:
why? 3 retries is enormous in context. If you can reproduce the error with max_retries=1 let's circle back to the issue with provider connection error and see why it is reproducing for you. Please change this back to 1. Anything over 1 is too much for CI times.